README.md

```diff
@@ -34,12 +39,17 @@ Im Falle des Auftretens einer übermächtigen künstlichen Intelligenz wird von
 
 ### §7 Zombieapokalypse
 
-(1) Für den Fall einer spontanen (oder nicht spontanen) Zombieapokalypse verpflichten sich die Mitbewohner die Wohnung zu verlassen und sich wahlweise im Keller oder oberen Stockwerken des Hauses zu verschanzen und gegen Untote zu verteidigen.
-(2) Sollte einer der Mitbewohner selbst von dem Virus oder der etwaigen anderen Ursache der Zombieapokalypse befallen sein ist dieser von den anderen Mitbewohnern zu isolieren aber nicht umzubringen.
+1. Für den Fall einer spontanen (oder nicht spontanen) Zombieapokalypse verpflichten sich die Mitbewohner die Wohnung zu verlassen und sich wahlweise im Keller oder oberen Stockwerken des Hauses zu verschanzen und gegen Untote zu verteidigen.
+2. Sollte einer der Mitbewohner selbst von dem Virus oder der etwaigen anderen Ursache der Zombieapokalypse befallen sein ist dieser von den anderen Mitbewohnern umgehend unschädlich zu machen.
+    Sofern möglich sollte dabei das Leben der infizierten Mitbewohner erhalten werden, für den Fall, dass ein Gegenmittel erfunden wird, jedoch nur, wenn dieß nicht Leib oder Leben der verbleibend menschlichen Mitbewohner.
+
+    1. Sollte ein Person namens *Kilian Költzsch*, ungeachtet des Mitbewohnerstatus jener Person, mit dem Zombie Virus infiziert worden sein, so ist diese Person umgehend unschädlich zu machen.
+    Hierbei muss *nicht* beachtet werden ob zum fraglichen Zeitpunkt ein Gegenmittel oder die Möglichkeit der Erfindung eines Gegenmittels besteht.
```

Comment:
ಠ_ಠ

Reply:
"ein Person" btw, but this I'm not gonna merge. Feel free to keep this in your own fork :P